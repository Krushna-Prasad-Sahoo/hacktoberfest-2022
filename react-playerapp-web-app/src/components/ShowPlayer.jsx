import React, { useContext, useEffect, useState } from 'react'
// import {PlayerContext} from '../data/PlayerContext';
import axios from 'axios';
export default function ShowPlayer() {
  // initialise the player info using useContext hook
  // const user = useContext(PlayerContext);
  // console.log("user2" , user)
  // useEffect(async ()=>{
  const [data, setData] = useState([]);
  useEffect(async () => {
    const res = await axios.get('http://localhost:3000/players');
    setData(res.data);

  }, [])



  // })
  return (
    <div class="data-table">
      <table>
        <tr>
          <th>
            Player Name
          </th>

          <th>
            Country
          </th>
          <th>
            Matches Played
          </th>
        </tr>



        {
          data && data.map((item, err) => {
            return (<>
              <tr>
                <td>{item.name}</td>
                <td>{item.country}</td>
                <td>{item.matchesPlayed}</td>
              </tr>

            </>)
          })
        }

      </table>
    </div>
  )
}
