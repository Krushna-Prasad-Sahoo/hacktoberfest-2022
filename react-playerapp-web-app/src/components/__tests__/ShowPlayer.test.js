
import toJson from 'enzyme-to-json';
import {useContext} from 'react';
import Enzyme, { shallow, render, mount } from 'enzyme';
import ShowPlayer from '../ShowPlayer';
xdescribe("Show Player Testing",()=>{
    const wrapper = shallow(<ShowPlayer/>) 
it('renders correctly Showplayer', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
   
    
})