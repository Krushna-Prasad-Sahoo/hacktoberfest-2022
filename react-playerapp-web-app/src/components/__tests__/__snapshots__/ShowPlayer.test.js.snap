// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show Player Testing renders correctly Showplayer 1`] = `undefined`;
